src/nrepl/index.ts

```diff
@@ -381,6 +382,7 @@ export class NReplSession {
           if (evaluation.onMessage(msg, pprintOptions)) {
             return true;
           }
+          return false;
```

Comment:
I'm unsure of this behavior but we must return a boolean from this function and it clearly shouldn't return true.